package.json

```diff
@@ -19,6 +19,9 @@
     "compile": "rm -rf ./target && node ../../scripts/build_kibana_platform_plugins.js --scan-dir . --filter enhancedTable"
```

Comment:
Could you replace this line by:
`"compile": "rm -rf ./target && node ../../scripts/plugin_helpers.js build --kibana-version 7.16.0 --skip-archive && mv build/kibana/enhancedTable/target . && rm -rf build/*"`

After test, I notice that, this way, generated "target" folder is 4x smaller.

Reply:
Are you ok with "--kibana version 7.16.0" being hardcoded? Does this work too for 7.17?

Reply:
Well, actually, version is useful only for "kibana.json" file.
If you look into "target" directory, there is nothing that references "7.16.0" version.
"kibana-version" is a required parameter, so I must pass a value, but value is not important, given that I only keep "target" directory.
All of that is done to generate only one time "target" directory, and then generate several zip files (and so versions) for the same target directory. And this is done because "target" directory generation is quite long.

That said, to remove any ambiguity, finally replace "7.16.0" by "none".
Thus, like that, for future versions (8.0 for example), I won't change this value for each branch.
Could you do this last change?

Reply:
I made the change you suggested